en/docs/get-started/about-this-release.md

```diff
@@ -12,227 +12,58 @@ For more information on WSO2 API Manager, see the [overview]({{base_path}}/getti
 
 ## New features
 
-#### API Governance
+??? note "JDK 17 support"
+    You can now run our products with JDK 17
 
-??? note "Management of real-time APIs on Solace"
+??? note "OpenTelemetry Support"
+    OpenTelemetry combines OpenTracing and OpenCencus which is a single set of APIs and libraries that standardize how telemetry data such as traces, metrics and logs are collected, transmitted and managed. It offers a safe, vendor-neutral specification for instrumentation so that information about the selection can be communicated to various backends
 
-    WSO2 API Manager 4.1.0 now supports integrating external third-party Gateways and Brokers. Solace is an advanced event broker that is efficiently streaming events and information across cloud, on-premises, and IoT environments.
+??? note "ELK Support for MI"
+    From WSO2 API Manager 4.2.0 onwards, Micro Integrator now enables publishing operational analytics for Elastic stack as a replacement for WSO2 EI Analytics.
 
-    **[Learn more]({{base_path}}/tutorials/integrating-with-solace/)**
+??? note "Configuring Self Sign up through Admin Portal"
+    WSO2 API Manager now offers you the ability to self-signup into the Developer Portal as well as the option to disable self-signup. You can also migrate the self-signup configurations from the registry through the **Advance Configuration** option or through editing the `tenant-conf.json` file.
```

Comment:
This should change a little. The self signup capability is already there in the Devportal. What comes in as a new feature is the ability to configure self signup through the admin portal as opposed to the configuration we did from the Management console.